### PR TITLE
Cloudfront Dashboards

### PR DESCRIPTION
**Pull request by rohitsinglagithub**
*Issue #, if available:* Yaml templates for Cloudfront Dashboards

*Description of changes:* Yaml templates for Cloudfront Dashboards

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: dashboards/cloudfront-dashboard-templates/cloudfront_realtime_logs-analysis.yaml [hunk 1]
```diff
@@ -0,0 +1,195 @@
+dashboards:
+  CLOUDFRONT_REALTIME_LOGS ANALYSIS:
+    name: cloudfront_realtime_logs analysis
```
Comment (iakov-aws) [inline dashboards/cloudfront-dashboard-templates/cloudfront_realtime_logs-analysis.yaml:3]: why analysis?

Comment (rohitsinglagithub) [reply dashboards/cloudfront-dashboard-templates/cloudfront_realtime_logs-analysis.yaml:3]: this seems to have taken the name of analysis while exporting using cid-cmd. can i change it manually without breaking the template?

Comment (iakov-aws) [reply dashboards/cloudfront-dashboard-templates/cloudfront_realtime_logs-analysis.yaml:3]: yes

Comment (rohitsinglagithub) [reply dashboards/cloudfront-dashboard-templates/cloudfront_realtime_logs-analysis.yaml]: updated both files.

Comment (iakov-aws) [reply dashboards/cloudfront-dashboard-templates/cloudfront_realtime_logs-analysis.yaml]: name is something that customer see as title 

CloudFront Realtime Logs

dahsbboard id is url so dashes are ok

BTW is it real RealTime?

Comment (rohitsinglagithub) [reply dashboards/cloudfront-dashboard-templates/cloudfront_realtime_logs-analysis.yaml]: Realtime - Logs are realtime, dashboard refresh, the fastest is 15 mins based on dataset refresh time. Exploring ways to speed it up further.

**Review by iakov-aws — APPROVED** [review]

**Pull request merged**